src/styles/_markdown-theme.scss

```diff
@@ -17,5 +17,25 @@
     background: rgba(mat-color($foreground, secondary-text), .01);
     border: .5px solid rgba(mat-color($foreground, secondary-text), .03);
   }
+
+  .docs-markdown-h3,
+  .docs-markdown-h4 {
+    position: relative;
+    padding-left: 30px;
+    margin-left: -30px;
+
+    .material-icons {
+      display: none;
+    }
+
+    &:hover {
+      .material-icons {
```

Comment:
Looks like this is meant to target the header link but is more broad than that. Also why does it need to be absolute?